lib/components/Tag/Tag.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ReactNode } from 'react';
+import { useStyles } from 'sku/treat';
+import { Box } from '../Box/Box';
+import { Text } from '../Text/Text';
+import { ClearButton } from '../iconButtons/ClearButton/ClearButton';
+import * as styleRefs from './Tag.treat';
+
+export interface TagProps {
+  onClear?: () => void;
+  clearLabel?: string;
```

Comment:
Maybe we should enforce a clearLabel?

Reply:
Done. It's now required when `onClear` is provided.